smpte.js

```diff
@@ -612,6 +612,11 @@ function insertConformance(docMetadata) {
     return;
   }
 
+  if (docMetadata.pubType == smpte.EG_PUBTYPE) {
+    if (sec !== null)
```

Comment:
```suggestion
```

Reply:
Updated in https://github.com/SMPTE/html-pub/pull/287/commits/212bd78f67077d21a0bbed3745d62b46f3ada8ab